CCPAPI/PAPI/ccTrace.h

```diff
@@ -0,0 +1,31 @@
+//##########################################################################
+//#                                                                        #
+//#                               CC                                       #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU Library General Public License as       #
+//#  published by the Free Software Foundation; version 2 or later of the  #
+//#  License.                                                              #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+//#                                                                        #
+//##########################################################################
+
+#ifndef CCPAPI_PAPI_CCTRACE_H_
+#define CCPAPI_PAPI_CCTRACE_H_
+
+#include <iostream>
+#include <sstream>
+
+#ifdef _CCDEBUG_
```

Comment:
Should this not be controlled by the build? i.e. whether we are building Debug or Release?